README.md

```diff
@@ -285,7 +286,8 @@ local list = {
   { key = "q",                            action = "close" },
   { key = "g?",                           action = "toggle_help" },
   { key = "W",                            action = "collapse_all" },
-  { key = "S",                            action = "search_node" }
+  { key = "S",                            action = "search_node" },
+  { key = "b",                            action = "run_file_command" }
```

Comment:
Minor:

I know it's late and it's another round of review, but personally I would rather have this mapping be under `.` to match vim-fugitive :smile: What do you think?

Reply:
I agree it would make more sense. `b` is not really a good default for `run command`. I'd have set something like `x` or `e` (execute), but `.` would be ok too.

Reply:
You could achieve that with creating a custom mapping when using nvim-tree :P

`{key = '.', action = 'run_file_command'}`

Reply:
what we mean here is having a default like this would make more sense :smile:

Reply:
I was just joking ;) I've changed it to `.`